tools/WinObjC.Tools/WinObjC.Tools.nuproj

```diff
@@ -14,19 +14,18 @@
     </ProjectConfiguration>
   </ItemGroup>
   <PropertyGroup Label="Configuration">
-    <PackageId>WinObjC.Tools</PackageId>
-    <Title>WinObjC.Tools</Title>
+    <PackageId>winobjc-tools</PackageId>
+    <Title>winobjc-tools</Title>
     <Authors>Microsoft</Authors>
     <Owners>Microsoft</Owners>
-    <Summary>WinObjC.Tools</Summary>
-    <Description>WinObjC.Tools</Description>
+    <Summary>Command Line tools and Visual Studio Extensions for the WinObjC Project.</Summary>
+    <Description>Command Line tools and Visual Studio Extensions for the WinObjC Project. This package includes vsimporter, the tool to convert an existing xcode objective c project to a visual studio solution, as well as the Visual Studio Extensions for Objective C syntax highlighting and Nuget package creation. See https://github.com/Microsoft/WinObjC for more details.</Description>
     <PackageReleaseNotes>
     </PackageReleaseNotes>
     <PackageProjectUrl>https://github.com/Microsoft/WinObjC</PackageProjectUrl>
-    <PackageLicenseUrl>
-    </PackageLicenseUrl>
+    <PackageLicenseUrl>https://github.com/Microsoft/WinObjC/blob/develop/license.txt</PackageLicenseUrl>
```

Comment:
I was thinking develop since it will be the latest always. In case we have changes, this will most quickly reflect any license changes but I honestly don't know which is better.

---
In reply to: [106000391](https://github.com/Microsoft/WinObjC/pull/2233#discussion_r106000391) [](ancestors = 106000391)